src/ec_asn1.c

```diff
@@ -0,0 +1,214 @@
+/*
+ * ASN.1 DER parsing
```

Comment:
Just out of interesst? From where is the ASN1 code derived from?

Reply:
The ASN.1 code comes from https://w1.fi/ site.